classes/ObjectModel.php

```diff
@@ -1235,6 +1235,8 @@ public static function displayFieldName($field, $class = __CLASS__, $htmlentitie
      * @param bool $htmlentities if true, uses htmlentities() for field name translations in errors
      *
      * @return array
+     *
+     * @throws \PrestaShop\PrestaShop\Adapter\CoreException
```

Comment:
This method throws nothing here :thinking: 
If you start to use the `@throws` you need to specify the reason as it's described in the PHPdoc documentation
> This tag is used to present in your documentation which error COULD occur and under which circumstances. It is RECOMMENDED to provide a description that describes the reason an exception is thrown.

Otherwise adding a `@throws` here is in my own opinion useless

Reply:
@PierreRambaud To be honest, I did not add these `@throws`. My IDE did 😄 when I was working on these functions I just committed the result.

What you say makes sense 👍 but my IDE tells me these `throws` are missing, not the context 😆 . I will close my PR then.